Cartfile

```diff
@@ -1 +1 @@
-github "VirgilSecurity/virgil-cryptowrapper-x" ~> 0.10.0
\ No newline at end of file
+github "VirgilSecurity/virgil-cryptowrapper-x" "develop"
```

Comment:
It's bad to have dependency on develop branch. Why not release cryptowrapper with needed changes?